### DISCUSSION TITLE
About mali gpu

### DISCUSSION THREAD

**Discussion by Anselshiu** [skyline-emu/skyline#1704] [Ideas]
Finally the devs are working on mali.But the performance in 3d games still not improved.
Here's a suggestion to do is that work on pokemong let's go first at least let us don't force freeze at the choose controller menu and next work on kamen rider games lag freeze and even force close in game.

**Comment by dima-xd** [comment]
Thanks for suggestion. Devs are starting work on pokemon. It's the main goal now

**Reply by Anselshiu** [reply]
Thanks for replying and I am really happy to hear this good news

**Comment by Anselshiu** [comment]
Another thing is a good news that kamen rider memory of heroes can load more but still very very lag and crash
